CONTRIBUTING.md

```diff
@@ -29,6 +29,48 @@ To contribute a change, please:
 6. If changes are requested, the PR author should address the comments, and then
    ask for review again.
 
+### Developing the cookiecutter template
+
+We quite like installing cookiecutter as a uv tool:
+
+```sh
+uv tool install cookiecutter
+```
+
+To run cookiecutter using a specific branch of the template:
+
+```sh
+cookiecutter gh:UCL-ARC/python-tooling --checkout <branch-name>
+```
+
+If you have this repo locally (presumably the case if you're developing), you
+can run the following:
+
+```sh
+cookiecutter /path/to/your/checkout/of/python-tooling --checkout <branch-name>
+```
+
+Perhaps obvious, you can omit the `--checkout` option if you're already on the
+branch you want to test.
+
+### Developing the recommended tooling pages
+
+Pages all live in the
+[docs/pages](https://github.com/UCL-ARC/python-tooling/tree/main/docs/pages)
```

Comment:
```suggestion
`docs/pages`
```
I'm not sure what having a link to the GH repo adds here?

Reply:
Yeah, good point. I think this is how it was in the README. 

🤷